src/query/storages/fuse/src/operations/update.rs

```diff
@@ -257,8 +270,7 @@ impl FuseTable {
         let (parts, part_info) = self
             .do_mutation_block_pruning(
                 ctx.clone(),
-                filter,
-                None,
+                filters,
```

Comment:
In the original codes, the inverted filter is `None`. However, it is changed to `Some` here.

Reply:
here we combine it into one struct named `Filters`.

`delete.rs` also calls this function but it need to pass  the inverted filter

Reply:
cc @JackTan25

Reply:
it's well, the update won't use this, it will do "read_pruning", and update doesn't need inversed filter, because if one block need to be updated ,we will read the whole block, but for delete, we don't need to do this, just give a delete log for commit meta.